docs/source/usage.rst

```diff
@@ -0,0 +1,164 @@
+.. _usage:
+
+Using ``repo2docker``
+=====================
+
+The core feature of repo2docker is to fetch a repo (from github or locally),
+build a container image based on the specifications found in the
+repo & optionally launch a local Jupyter Notebook you can use to explore it.
+
+This section describes the general ways in which you can use
+``repo2docker``. It covers basics in how to prepare your
```

Comment:
This sections describes common tasks when using ``repo2docker`` including:

- Preparing your repository
- Running repo2docker locally
- Displaying a generated Dockerfile
- Accessing help from the command line